recipes/expat-cos7-aarch64/meta.yaml

```diff
@@ -0,0 +1,73 @@
+# A few variables that make this a tiny bit more portable
+{% set name = "expat" %}
```

Comment:
I don't see why we'd need a CDT for this. There's a `expat` package in conda-forge.

Reply:
Truthfully, i don't fully understand what the best way to go about packaging X11 is.

https://github.com/conda-forge/gstreamer-feedstock/blob/master/recipe/meta.yaml#L82

Some people use CDTs, others use the X11 package. Seems a little random, and definitely "whatever works".

Reply:
I understand arguments for both CDTs and packages for X11. But why expat?

Reply:
From the comment above, I guess it is a dependency for the `mesa-dri-drivers`
```
# expat here is _only_ required for mesa-dri-drivers
```

Reply:
Not sure if this build error makes sense to you. It does a little to me :/
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=61433&view=logs&jobId=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&taskId=841356e0-85bb-57d8-dbbc-852e683d1642&lineStart=16277&lineEnd=16278&colStart=1&colEnd=1

Reply:
well, you have to add `expat` conda package if you remove the CDT

Reply:
Right, but I think the idea was that that since it was a dependency for a CDT, it should be added as a CDT itself right?

Reply:
@isuruf Any thoughts?

Reply:
>  it should be added as a CDT itself right?

Okay